lib/lita/handlers/karma/chat.rb

```diff
@@ -104,7 +104,7 @@ def delete(response)
 
     def define_dynamic_routes(pattern)
       self.class.route(
-        %r{(#{pattern})\+\+#{token_terminator.source}},
+        %r{(#{pattern}) {0,1}\+\+#{token_terminator.source}},
```

Comment:
Lol - my mind went down the exact same route; and came to the same conclusion "let's just start with spaces"